### ISSUE TITLE
How to compile and run it with IOS7

### ISSUE THREAD

**Issue by naveens2si** [lukeweber/webrtc-jingle-client#115]
Hello i am trying to build a IOS7 app. 
I followed all your procedure and the xcodeproj file has been generated.
Once i open it using Xcode and compile it and run the app crashes.
Any one successfully compiled webrtc jingle for IOS7?
Any help or direction would be useful.

**Comment by McPo** [comment]
I can't provide details (I don't know them), but can confirm its been compiled and running correctly on iOS7. Although I wouldnt know if the sample app (if one exists) works.

**Comment by naveens2si** [comment]
Is there any working sample app or anyone who can help me regarding this?

**Comment by chathudan** [comment]
did you change the log in  details correctly ?

**Comment by naveens2si** [comment]
Yes i changed but the problem is it is not even running it give more than 100 errors in my Xcode.

**Comment by McPo** [comment]
Post the errors

**Comment by ahndee** [comment]
Remove the *.storyboard files from the project navigator and drag beck back from the Finder - that most likely will fix things
